magento1-module/app/code/local/Divante/VueStorefrontBridge/controllers/UserController.php

```diff
@@ -321,29 +323,61 @@ public function meAction(){
                         foreach($updatedCustomer['addresses'] as $updatedAdress) {
                             $updatedAdress['region'] = $updatedAdress['region']['region'];
 
-                            if($updatedAdress['default_billing']) {
+                            if ($updatedAdress['delete'] === true) {
+                                try {
+                                    $sAddress = Mage::getModel('customer/address');
+
+                                    $sAddress->load($updatedAdress['entity_id']);
+
+                                    $sAddress->setData($updatedAdress)->delete();
+                                } catch (Throwable $exception) {
+                                    Mage::log(
```

Comment:
Please log the exception message as well for easier debugging

Reply:
Added it.